### DISCUSSION TITLE
Welcome to -TO-DO-List-Plain Discussions!

### DISCUSSION THREAD

**Discussion by Thakor-Yashpal** [Thakor-Yashpal/-TO-DO-List-Plain#4] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Thakor-Yashpal** [comment, marked as answer]
any one can help ?

**Reply by YASH-T-0** [reply]
what kind of  help you need ?

**Reply by Thakor-Yashpal** [reply]
nothing

**Reply by YASH-T-0** [reply]
published this

**Reply by Thakor-Yashpal** [reply]
ok

**Reply by YASH-T-0** [reply]
> published this

yes

**Comment by YASH-T-0** [comment]
you can ?

**Reply by Thakor-Yashpal** [reply]
what is it ?

**Reply by Thakor-Yashpal** [reply]
> you can ?

 q
